README.md

```diff
@@ -132,8 +132,9 @@ spec:
 ```
 
 ### Configuring OVNKubernetes
-OVNKubernetes supports the following configuration options, all of which are optional:
+OVNKubernetes supports the following configuration options, all of which are optional and once set at cluster creation, they can't be changed:
 * `MTU`: The MTU to use for the geneve overlay. The default is the MTU of the node that the cluster-network-operator is first run on, minus 100 bytes for geneve overhead. If the nodes in your cluster don't all have the same MTU then you may need to set this explicitly.
+* `genevePort`: The UDP port to use for the Geneve overlay. The default is 6081, on OpenShift the default is 9081.
```

Comment:
I thought we were changing the nbdb and sbdb ports but leaving the Geneve port unchanged, for each of tcpdump debugging.

Reply:
The sdb and ndb ports changed in PR 313. This also shifts the geneve port to 9000-9999.

Reply:
My understanding is that we had explicitly decided that we didn't want to change the default geneve port.

Reply:
@squeed What is your desire in changing the geneve port. See @danwinship comment above.

Reply:
Will use 6081 and allow customer to change port.